spinoso-string/src/enc/binary/impls.rs

```diff
@@ -0,0 +1,180 @@
+use alloc::borrow::Cow;
+use alloc::string::String;
+use alloc::vec::Vec;
+use core::borrow::{Borrow, BorrowMut};
+use core::fmt::Arguments;
```

Comment:
It looks like we need one of these:

```suggestion
#[cfg(feature = "std")]
use core::fmt::Arguments;
```